examples/tutorials/nb_python/managed_rigid_object_tutorial.py

```diff
@@ -0,0 +1,560 @@
+    print("Simulating " + str(dt) + " world seconds.")
+    observations = []
+    start_time = sim.get_world_time()
+    while sim.get_world_time() < start_time + dt:
+        sim.step_physics(1.0 / 60.0)
+        if get_frames:
+            observations.append(sim.get_sensor_observations())
+
+    return observations
+
+
+# [/setup]
+if __name__ == "__main__":
+    import argparse
+
+    parser = argparse.ArgumentParser()
+    parser.add_argument("--no-show-video", dest="show_video", action="store_false")
+    parser.add_argument("--no-make-video", dest="make_video", action="store_false")
+    parser.set_defaults(show_video=True, make_video=True)
+    args, _ = parser.parse_known_args()
+    show_video = args.show_video
+    make_video = args.make_video
+    if make_video and not os.path.exists(output_path):
+        os.mkdir(output_path)
+
+    # %%
+    # [initialize]
+    # create the simulators AND resets the simulator
+
+    cfg = make_configuration()
+    try:  # Got to make initialization idiot proof
+        sim.close()
+    except NameError:
+        pass
+    sim = habitat_sim.Simulator(cfg)
+    agent_transform = place_agent(sim)
+
+    # get the primitive assets attributes manager
+    prim_templates_mgr = sim.get_asset_template_manager()
+
+    # get the physics object attributes manager
+    obj_templates_mgr = sim.get_object_template_manager()
+
+    # get the rigid object manager
+    rigid_obj_mgr = sim.get_rigid_object_manager()
+    # [/initialize]
+
+    # %%
+    # [basics]
+
+    # load some object templates from configuration files
+    sphere_template_id = obj_templates_mgr.load_configs(
+        str(os.path.join(data_path, "test_assets/objects/sphere"))
+    )[0]
+
+    # add a sphere to the scene, returns the object
+    sphere_obj = rigid_obj_mgr.add_object_by_id(sphere_template_id)
+    # move sphere
+    sphere_obj.translation = np.array([2.50, 0, 0.2])
+
+    # simulate
+    observations = simulate(sim, dt=1.5, get_frames=make_video)
+
+    if make_video:
+        vut.make_video(
+            observations,
+            "rgba_camera_1stperson",
+            "color",
+            output_path + "sim_basics",
+            open_vid=show_video,
+        )
+
+    # [/basics]
+    rigid_obj_mgr.remove_all_objects()
+    # %%
+    # [dynamic_control]
+
+    observations = []
+    obj_templates_mgr.load_configs(str(os.path.join(data_path, "objects")))
+    # search for an object template by key sub-string
+    cheezit_template_handle = obj_templates_mgr.get_template_handles(
+        "data/objects/cheezit"
+    )[0]
+    # build multiple object initial positions
+    box_positions = [
+        np.array([2.39, -0.37, 0]),
+        np.array([2.39, -0.64, 0]),
+        np.array([2.39, -0.91, 0]),
+        np.array([2.39, -0.64, -0.22]),
+        np.array([2.39, -0.64, 0.22]),
+    ]
+    box_orientation = mn.Quaternion.rotation(
+        mn.Rad(math.pi / 2.0), np.array([-1.0, 0, 0])
```

Comment:
```suggestion
        mn.Deg(90.0), np.array([-1.0, 0, 0])
```

Works the same as in C++, degrees and radians convert implicitly to each other.